### PR TITLE
create_order.scooter

### PR DESCRIPTION
**Pull request by avobl**

### CODE REVIEW

File: services/grpcscooter.go [hunk 1]
```diff
@@ -58,7 +57,7 @@ func (gss *GrpcScooterService) InitAndRun(scooterID int,
 		return err
 	}
 
-	if scooterStatus.BatteryRemain > 10 {
+	if scooterStatus.BatteryRemain > 10 && scooter.CanBeRent == true {
```
Comment (DmytroTHR) [inline services/grpcscooter.go]: no need to write "==true"

Comment (avobl) [reply services/grpcscooter.go]: Ohhh, sure. Thanks ;)

File: repositories/order.go [hunk 2]
```diff
@@ -3,8 +3,11 @@ package repositories
 import "Dp218Go/models"
 
 type OrderRepo interface {
-	CreateOrder(user models.User, scooter models.ScooterDTO) (models.Order, error)
-	SetOrderStart(order *models.Order, status models.ScooterStatusInRent) error
-	SetOrderEnd(order *models.Order, status models.ScooterStatusInRent) error
-	UpdateOrder(order *models.Order) error
+	CreateOrder(user models.User, scooterID, startID, endID int, distance float64) (models.Order, error)
+	GetAllOrders() (*models.OrderList, error)
+	//SetOrderStart(order *models.Order, status models.ScooterStatusInRent) error
```
Comment (DmytroTHR) [inline repositories/order.go]: GetOrderByID() should also be very useful. In fact it's the only func, that I need for my purposes))

File: repositories/postgres/postgres.go [hunk 3]
```diff
@@ -12,7 +12,7 @@ import (
 )
 
 const (
-	defaultMaxPoolSize  = 1//3
+	defaultMaxPoolSize  = 5//3
```
Comment (DmytroTHR) [inline repositories/postgres/postgres.go]: I made it 3 in my PR. Probably, let's try  from 3 for not having merge conflicts here?

File: routing/scooter.go [hunk 4]
```diff
@@ -11,14 +11,16 @@ import (
...
 // TODO make dynamic access to this variables from UI
 //var choosenWay = models.Coordinate{Latitude: 48.4221, Longitude: 35.0196}
-//var choosenWay = models.Coordinate{ Latitude: 48.42543, Longitude: 35.02183}  // dafi
+var choosenWay = models.Coordinate{Latitude: 48.42543, Longitude: 35.02183} // dafi
 //var choosenWay = models.Coordinate{48.42272,35.02280} // visokovoltnaya
-var choosenWay = models.Coordinate{Latitude: 48.42367 , Longitude: 35.04436} // ostapa vishni
-var choosenScooter = 1
+//var choosenWay = models.Coordinate{Latitude: 48.42367 , Longitude: 35.04436} // ostapa vishni
+var choosenScooterID = 1
+var userFromRequest = models.User{ID: 1, LoginEmail: "guru_admin@guru.com", UserName: "Guru", UserSurname: "Sadh"}
```
Comment (DmytroTHR) [inline routing/scooter.go:23]: Did you try to get user from context? (the thing, that Alexey had shown us before)

**Pull request closed without merging**